Firmware/mmu2.cpp

```diff
@@ -44,6 +44,9 @@ static constexpr float MMU2_LOAD_TO_NOZZLE_LENGTH = 87.0F + 5.0F;
 // The printer intercepts such a call and sets its extra load distance to match the new value as well.
 static constexpr uint8_t MMU2_TOOL_CHANGE_LOAD_LENGTH = 5; // mm
 
+static constexpr uint8_t MMU2_EXTRUDER_PTFE_LENGTH = 42.3f; // mm
+static constexpr uint8_t MMU2_EXTRUDER_HEATBREAK_LENGTH  = 17.7f; // mm
```

Comment:
ummm... what's the point of specifying `.f` if it's getting downgraded to uint8_t?

Reply:
Omg, this was stupid of me.

Reply:
Fixed [df67c13](https://github.com/prusa3d/Prusa-Firmware/pull/3761/commits/df67c1384ca27243f9fe3eacadb256cff599dfe5)